src/transactions/InvokeHostFunctionOpFrame.cpp

```diff
@@ -497,6 +497,14 @@ InvokeHostFunctionOpFrame::doApply(Application& app, AbstractLedgerTxn& ltx,
         metrics.mCpuInsnExclVm = out.cpu_insns_excluding_vm_instantiation;
         metrics.mInvokeTimeNsecsExclVm =
             out.time_nsecs_excluding_vm_instantiation;
+
+        CLOG_TRACE(Tx, "invokeTimeNsecs: {}, cpuInsn: {}, ratio: {}",
+                   metrics.mInvokeTimeNsecs, metrics.mCpuInsn,
+                   (float)metrics.mInvokeTimeNsecs / (float)metrics.mCpuInsn);
+        CLOG_TRACE(Tx, "(excl VM) invokeTimeNsecs: {}, cpuInsn: {}, ratio: {}",
+                   metrics.mInvokeTimeNsecsExclVm, metrics.mCpuInsnExclVm,
+                   (float)metrics.mInvokeTimeNsecsExclVm /
+                       (float)metrics.mCpuInsnExclVm);
```

Comment:
Nit: C++ Style cast. We also only need to cast one of the numerator or denominator. Finally since this is just logs, why not use double for extra precision?